bambi/models.py

```diff
@@ -678,22 +674,23 @@ def plot_priors(self, varnames=None):
                     continue
                 prior = t.prior.args['sd'].name
                 params = t.prior.args['sd'].args
-                dists += [getattr(pm, prior)(t.name+'_sd', **params)]
+                dists += [getattr(pm, prior)(t.name + '_sd', **params)]
 
             # add priors on Y params if applicable
             y_prior = [(k, v) for k, v in self.y.prior.args.items()
                        if isinstance(v, Prior)]
             if len(y_prior):
                 for p in y_prior:
-                    dists += [getattr(pm, p[1].name)('_'.join([self.y.name,
-                                                               p[0]]), **p[1].args)]
+                    pm_attr = getattr(pm, p[1].name)
+                    p = pm_attr('_'.join([self.y.name, p[0]]), **p[1].args)
+                    dists.extend([p])
```

Comment:
Nice readability edits. Also I finally installed a linter, so in theory I'll be avoiding some of the patterns above (like no space around operators) in the future, and will probably fix these kinds of things that tick off the linter as I come across them in the codebase :)